runtime/nvqir/cutensornet/CMakeLists.txt

```diff
@@ -0,0 +1,88 @@
+find_package(CUDA REQUIRED)
+find_package(CUDAToolkit REQUIRED)
```

Comment:
Great question. Actually, we only need the `CUDAToolkit` in this case, not `CUDA`, since we don't have any CUDA kernels here. Plus, `find_package(CUDA)` is deprecated as per CMake [docs](https://cmake.org/cmake/help/v3.20/module/FindCUDA.html). Removed the `find_package(CUDA)` line.